skate-plugin/build.gradle.kts

```diff
@@ -26,6 +26,8 @@ plugins {
   alias(libs.plugins.pluginUploader)
   alias(libs.plugins.buildConfig)
   alias(libs.plugins.lint)
+  alias(libs.plugins.compose)
+  alias(libs.plugins.kotlin.plugin.compose)
```

Comment:
let's move this to a separate subproject like we did with the project generator, we had issues in the past getting this to stay working nice. You can add it to the project generator too and just give the project a more generic name, like skate-ui or something?

Reply:
Does this mean moving everything with aibot into a separate project? And what do you mean by adding it to project generator too?

Reply:
yeah I meant not having compose code directly in the `skate-plugin` project sources, we've struggled to keep that build setup working in the past. The project generator (which is also compose UI) is in a separate subproject, I'm suggesting you can move it there

Reply:
Oh okay, that makes sense. I can rename the project-gen subproject to skate-ui and then add the aibot src code into that instead of skate-plugin